### PR TITLE
Greatly optimized code

### PR DESCRIPTION
**Pull request by sourTaste000**
tl;dr deleted a lot of useless code

### CODE REVIEW

**Comment by sourTaste000** [comment]
also let css to do the 404 page, not an image.

File: 404.html [hunk 1]
```diff
@@ -3,37 +3,21 @@
...
-	<meta content="https://kamiblue.org/404" property="og:url">
-	<meta content="KAMI Blue" property="og:title">
-	<meta content="Error 404" property="og:description">
-	<meta content="/assets/images/logoSmall.png" property="og:image">
-	<!-- Twitter Meta Tags -->
-	<meta content="summary_large_image" property="twitter:card">
-	<meta content="https://kamiblue.org/404" property="twitter:url">
-	<meta content="KAMI Blue" property="twitter:title">
-	<meta content="Error 404" property="twitter:description">
-	<meta content="/assets/images/logoSmall.png" property="twitter:image">
```
Comment (dominikaaaa) [inline 404.html:24]: we previously had these because they improve SEO

File: assets/404/style.css [hunk 2]
```diff
@@ -1,105 +1,41 @@
...
+	url('https://cdn.jsdelivr.net/gh/JetBrains/JetBrainsMono/ttf/JetBrainsMono-Regular.ttf') format('truetype');
+	font-weight: 400;
+	font-style: normal;
+}
+
 body {
 	background: #191919;
 	color: #ffffff;
-	font-family: 'Fira Code', monospace;
+	font-family: "JetBrains Mono", "Fira Code Retina", Courier, monospace;
```
Comment (dominikaaaa) [inline assets/404/style.css:17]: Fira Code Retina should be first

**Review by dominikaaaa — CHANGES_REQUESTED** [review]

**Comment by humboldt123** [comment]
Please make the changes @dominikaaaa suggested.

**Comment by dominikaaaa** [comment]
We should be using templates per each page. I'll make that change myself

File: assets/404/style.css (hunk 2 above)
Comment (dominikaaaa) [reply assets/404/style.css:17]: Also I'm not sure if it'll work, looking at the original file. https://github.com/tonsky/FiraCode/issues/1126

**Comment by humboldt123** [comment]
Good idea.

File: assets/index/script.js [hunk 3]
```diff
@@ -245,7 +245,7 @@ const devDiv =
     <div>
       Please make sure to restart your IDE and reimport 🔄 gradle after running all the setup
       <br>Read Support for help and Troubleshooting for common errors. TODO TODO
-      <br>When making a pull request or issue, please follow the Code of Conduct TODo
+      <br>When making a pull request or issue, please follow the Code of Conduct
```
Comment (dominikaaaa) [inline assets/index/script.js:248]: this is todo tho

**Review by dominikaaaa — CHANGES_REQUESTED** [review]

**Comment by sourTaste000** [comment]
bruh conflict be like

**Comment by humboldt123** [comment]
I recommend copying the files, manually committing, then making a pull.

**Pull request closed without merging**